web/vtadmin/src/components/routes/tablets/InfoDropdown.tsx

```diff
@@ -1,25 +1,67 @@
 import React, { useState } from 'react';
+import { useHealthCheck, usePingTablet, useRefreshState } from '../../../hooks/api';
 import Dropdown from '../../dropdown/Dropdown';
 import MenuItem from '../../dropdown/MenuItem';
 import { Icons } from '../../Icon';
-import PingDialog from './PingDialog';
-
+import InfoDialog, { BaseInfoDialogProps } from './InfoDialog';
 interface InfoDropdownProps {
     alias: string;
     clusterID?: string;
 }
 
 const InfoDropdown: React.FC<InfoDropdownProps> = ({ alias, clusterID }) => {
-    const [isPingOpen, setPingOpen] = useState(false);
+    const [currentDialog, setCurrentDialog] = useState('');
```

Comment:
Is this...

```jsx
const [currentDialog, setCurrentDialog] = useState('ping');
```

...equivalent to this?

```jsx
const defaultConfig = dialogConfigs['ping'];
```

(Just curious if I'm missing a subtle thing with hooks!)